ai_edge_torch/generative/utilities/loader.py

```diff
@@ -158,7 +170,7 @@ def load(self, model: torch.nn.Module, strict: bool = True):
       raise ValueError(
           f"Failed to map all tensor. Remaing tensor are: {list(state.keys())}"
       )
-    model.load_state_dict(converted_state, strict=strict)
+    return model.load_state_dict(converted_state, strict=strict)
```

Comment:
Please update docstring and pytype annotations.

Reply:
Done